+p_xml: 'version="1.0" encoding="UTF-8" standalone="yes" '
+directive: DOCTYPE bugzilla SYSTEM "http://bugzilla.fieldtriptoolbox.org/page.cgi?id=bugzilla.dtd"
bugzilla:
  +@version: 4.4.1
  +@urlbase: http://bugzilla.fieldtriptoolbox.org/
  +@maintainer: r.oostenveld@donders.ru.nl
  bug:
    bug_id: "1052"
    creation_ts: 2011-10-18 15:25:00 +0200
    short_desc: fixsens breaks on EEG data
    delta_ts: 2011-11-11 11:19:33 +0100
    reporter_accessible: "1"
    cclist_accessible: "1"
    classification_id: "1"
    classification: Unclassified
    product: FieldTrip
    component: core
    version: unspecified
    rep_platform: PC
    op_sys: Windows
    bug_status: CLOSED
    resolution: DUPLICATE
    dup_id: "1053"
    bug_file_loc:
    status_whiteboard:
    keywords:
    priority: P1
    bug_severity: normal
    target_milestone: '---'
    everconfirmed: "1"
    reporter:
      +content: annevanhoogmoed
      +@name: Anne van Hoogmoed
    assigned_to: fieldtriptoolbox
    cc:
      - j.schoffelen
      - r.oostenveld
    comment_sort_order: oldest_to_newest
    long_desc:
      - +@isprivate: "0"
        commentid: "3585"
        comment_count: "0"
        who:
          +content: annevanhoogmoed
          +@name: Anne van Hoogmoed
        bug_when: 2011-10-18 15:25:56 +0200
        thetext: |-
          For some reason fixsens breaks on EEG data? probably related to previous fixsens bug?

          AFter this:

              trlnr = size(data_freq_odd_clean.trial,2);
              trlnew = data_freq_odd_clean.cfg.trl(1:3:trlnr,:);

              cfg = [];
              cfg.trl = trlnew;
              data_freq_3rd = ft_redefinetrial(cfg, data_freq_odd_clean);

          comes this:


          the input is raw data with 33 channels and 38 trials
          ??? Reference to non-existent field 'coilpos'.

          Error in ==> channelposition at 271
                pnt = sens.coilpos;

          Error in ==> fixsens at 61
            chanpos      = channelposition(sens, 'channel', 'all');

          Error in ==> ft_datatype_raw at 99
                data.elec = fixsens(data.elec);

          Error in ==> ft_checkdata at 175
            data = ft_datatype_raw(data, 'hassampleinfo', hassampleinfo);

          Error in ==> ft_redefinetrial at 115
          data = ft_checkdata(data, 'datatype', 'raw', 'feedback', cfg.feedback);
      - +@isprivate: "0"
        commentid: "3591"
        comment_count: "1"
        who:
          +content: r.oostenveld
          +@name: Robert Oostenveld
        bug_when: 2011-10-18 19:49:02 +0200
        thetext: '*** This bug has been marked as a duplicate of bug 1053 ***'
      - +@isprivate: "0"
        commentid: "3892"
        comment_count: "2"
        who:
          +content: r.oostenveld
          +@name: Robert Oostenveld
        bug_when: 2011-11-11 11:19:33 +0100
        thetext: I closed all bugs that were recently resolved.
